internal/github/teamMemList.go

```diff
@@ -0,0 +1,44 @@
+package github
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/google/go-github/v60/github"
+)
+
+// GetTeamMemList obtains a list of teammembers
+func GetTeamMemList(githubClient *github.Client, internalTeam string) (map[string]bool, error) {
```

Comment:
```suggestion
// GetTeamMemberList obtains a list of teammembers
func GetTeamMemberList(githubClient *github.Client, internalTeam string) (map[string]bool, error) {
```

More clear to just use full names. No penalties for a slightly longer function name here